apps/dashboard/app/(app)/settings/billing/change-plan-button.tsx

```
@@ -0,0 +1,130 @@
+              control={form.control}
+              name="productId"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>Choose new plan</FormLabel>
+                  <Select onValueChange={field.onChange} value={field.value || "none"}>
+                    <SelectTrigger>
+                      <SelectValue />
+                    </SelectTrigger>
+                    <SelectContent>
+                      {products.map((p) => (
+                        <SelectItem value={p.id} disabled={p.id === currentProductId}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)